nautobot_golden_config/api/filters.py

```diff
@@ -0,0 +1,36 @@
+"""Filters for API Views."""
+
+from nautobot_golden_config import models
+from nautobot_golden_config.filters import ConfigComplianceFilter
+
+
+class GoldenConfigFilter(ConfigComplianceFilter):
+    """Filter capabilities for GoldenConfig instances."""
+
+    class Meta:
+        """Meta class attributes for ConfigComplianceFilter."""
+
+        model = models.GoldenConfig
+        fields = [
+            "q",
+            "tenant_group_id",
+            "tenant_group",
+            "tenant_id",
+            "tenant",
+            "region_id",
+            "region",
+            "site_id",
+            "site",
+            "rack_group_id",
+            "rack_id",
+            "role_id",
+            "role",
+            "manufacturer_id",
+            "manufacturer",
+            "platform_id",
+            "platform",
+            "device_status_id",
+            "device_type_id",
+            "device_id",
+            "device",
+        ]
```

Comment:
If these fields are the same as those on the base `ConfigComplianceFilter` (it appears to me they are?), then I believe you could avoid repetition by inheritance of the Meta class, i.e.

```python
class Meta(ConfigComplianceFilter.Meta):
    model = models.GoldenConfig
```